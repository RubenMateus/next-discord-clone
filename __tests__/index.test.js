import { render } from "@testing-library/react";
import Home from "../pages/index";

describe("Home", () => {
  it("render snapshot", () => {
    const { container } = render(<Home />);

    expect(container).toMatchSnapshot();
  });
});