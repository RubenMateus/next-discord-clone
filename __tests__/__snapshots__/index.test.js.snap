// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home render snapshot 1`] = `
<div>
  <div
    class="flex flex-col bg-gray-800 w-60"
  >
    <div
      class="flex items-center h-12 px-3 font-semibold text-white shadow-md font-title"
    >
      Dashboard
    </div>
    <div
      class="flex-1 p-3 space-y-2 overflow-y-scroll font-medium text-gray-300 "
    >
      <p
        class="text-white"
      >
        Friends
      </p>
    </div>
  </div>
  <div
    class="flex flex-col flex-1 bg-gray-700"
  />
</div>
`;
